site/app/templates/polls/NewPollPage.twig

```diff
@@ -142,7 +142,7 @@
 <script>
     flatpickr('.poll-date', {
         allowInput: true,
-        dateFormat: "Y-m-d",
+        dateFormat: "m/d/Y",
```

Comment:
You can pass the date format from the view file, in line 51 of `site/app/views/PollView.php`
```php
return $this->core->getOutput()->renderTwigTemplate("polls/NewPollPage.twig", [
            'csrf_token' => $this->core->getCsrfToken(),
            'base_url' => $this->core->buildCourseUrl() . '/polls',
            'date_format' => $this->core->getConfig()->getDateTimeFormat()->getFormat('poll')
          ]);
```

Then in the NewPollPage.twig you can access it by {{ date_format }}

Here's a quick overview of Twig templating if you're not familiar 
https://twig.symfony.com/doc/3.x/templates.html

The connection between controllers, models, views, and twig templates might be confusing at first so feel free to ask in slack if anything isn't clear.